translate/src/context/BadgeTooltip.tsx

```diff
@@ -21,6 +21,26 @@ export function BadgeTooltipProvider({
 }) {
   const [message, setMessage] = useState<BadgeTooltipMessage | null>(null);
 
+  useEffect(() => {
+    const rootElt = document.getElementById('root');
+    if (rootElt?.dataset.notifications) {
+      const notifications = JSON.parse(rootElt.dataset.notifications);
+      if (notifications.length > 0) {
+        const parsed = notifications.map(
+          (notification: { content: string; type: string }) => {
+            if (notification.type === 'info') {
+              // Badge update information
+              return JSON.parse(notification.content);
+            } else {
+              return { content: notification.content, type: notification.type };
+            }
+          },
+        );
+        setMessage(parsed[1]);
+      }
+    }
+  }, []);
+
```

Comment:
Can we instead just modify the Notification context?

(Also, it's confusing that we refer to Django Messaging framework messages as Notifications (the name we already use for something completely different). Should possibly file a task to rename that.)

https://github.com/mozilla/pontoon/blob/a0b431cff0b6844dd1ba9018010db4010e11497e/translate/src/context/Notification.tsx#L25-L38

Reply:
Perhaps the right way forward here is to just factor out a function that checks if we have any notifications to display.